tests/test-dirs/destruct/dune

```diff
@@ -0,0 +1,3 @@
+(cram
+ (applies_to :whole_subtree)
+ (alias all-destruct-tests))
```

Comment:
why do we need this here, but for not in any of the other test dirs?

Reply:
We do not need it. It is just a convenient alias to run (and watch) all Destruct tests together.

Reply:
And just to confirm, it won't conflict in some way with the similar stanza in `tests/dune`? (And why don't we need to specify the test deps here?)

Reply:
In my understanding it adds with previous stanzas, not redefine them. But you make me doubt, I will ask Rudi.